app/src/org/commcare/utils/AndroidInstanceInitializer.java

```diff
@@ -56,6 +63,20 @@ protected AbstractTreeElement setupCaseData(ExternalDataInstance instance) {
         return casebase;
     }
 
+    @Override
+    protected AbstractTreeElement setupFixtureData(ExternalDataInstance instance) {
```

Comment:
@shubham1g5 We should make sure that we've also implemented anything we need to implement here for Formplayer. 

This may require, for example, require us to bump the version DB (which requires us to clear out Incomplete Forms). 

Unrelated to this PR structure, but useful for us to know whether we need to update the Formplayer DB Version # with the next commcare-core pull in that library before merging this one (let me know if this comment doesn't make sense).